src/dbsync/testtool/cmdArgsHelper.h

```diff
@@ -77,15 +77,15 @@ class CmdLineArgs
 
     static std::vector<std::string> splitActions(const std::string& values)
     {
-        std::vector<std::string> actions;
+        std::vector<std::string> actionList;
```

Comment:
according to the coding style it should be named actions instead of actionsList. pls revert this.

Reply:
I reverted to the previous name (actions). This will be fixed by @Dwordcito in his PR to fix the shadowing cppcheck issue correctly.